modules/deployment/variables.tf

```diff
@@ -150,6 +152,24 @@ variable "ecr_repository_name" {
   type        = string
 }
 
+variable "post_deployment_stages" {
```

Comment:
IMO we should prefix this with `codepipeline` since the module has a lot of vars (for CodePipeline, CodeDeploy et. al)

I try to keep the vars sorted lexicographically as well